docs/kmesh_compile-zh.md

```diff
@@ -137,6 +137,62 @@ Kmesh需要在拥有Kmesh内核增强特性的Linux环境中编译构建。当
   kmesh-0.0.1           latest              e321b18d5fee        4 hours ago         675MB
   ```
 
+### docker image [兼容模式](../build/docker/README.md)构建
+
+- 准备工作
+
+  - docker-engine安装
+
+    ```sh
+    [root@dev Kmesh]# yum install docker-engine
+    ```
+
+  - 镜像原料准备
+
+    Kmesh的镜像编译需要准备好kmesh源码、kmesh.dockerfile、start_kmesh.sh启动脚本；将其放在一个目录下；
+
+    kmesh.dockerfile、start_kmesh.sh归档在代码仓目录下：
+
+    ```sh
+    [root@dev Kmesh]# ll build/docker/
+    total 12K
+    -rw-r--r--. 1 root root  793 Nov 25 01:31 kmesh.dockerfile
+    -rw-r--r--. 1 root root 1.5K Nov 25 10:48 kmesh.yaml
+    -rw-r--r--. 1 root root  764 Nov 25 01:31 start_kmesh.sh
+    ```
+
+    将镜像原料放到一个目录下
+
+    ```sh
+    [root@dev docker]# ll
+    -rw-r--r--. 1 root root  793 Nov 25 01:36 kmesh.dockerfile
+    -rw-r--r--. 1 root root  764 Nov 25 01:36 start_kmesh.sh
+    -rw-r--r--. 1 root root  764 Nov 25 01:36 xxx
```

Comment:
can we provide a script doing this

Reply:
Do you mean that a script is provided to automatically build a kmesh image? In the future, we can invoke the script on the access control CI to build an image？